.github/workflows/build_and_test.yml

```diff
@@ -426,6 +428,7 @@ jobs:
       matrix:
         build_type: [ debug, release ]
         pg_version: [ v14, v15, v16 ]
+        pageserver_virtual_file_io_engine: [ std-fs, tokio-epoll-uring ]
```

Comment:
Unless we can expand the CI runner pool, I'll merge this PR with
```
pageserver_virtual_file_io_engine: [ std-fs ]
```

Shortly after the weekly release, I'll then switch it to 

```
pageserver_virtual_file_io_engine: [ tokio-epoll-uring ]
```

for 2-3 days, so that the tokio-epoll-uring code gets coverage.

Then I'll switch it back, so that in the days leading up to the subsequent release, we test with the IO engine we're using in production.

Reply:
I want to clean up the python part that I've added a bit.  
I've assigned myself as a reviewer, please don't merge without my approval 🙏